frontend/libs/studio-content-library/src/ContentLibrary/LibraryBody/pages/CodeListPage/CodeListPage.test.tsx

```diff
@@ -313,16 +313,15 @@ const getFirstDescriptionField = (area: HTMLElement): HTMLElement => {
   return within(area).getByRole('textbox', { name: descriptionFieldLabel });
 };
 
-const getCodeListAccordion = (codeListTitle: string): HTMLElement =>
-  // The following code accesses a node directly with parentElement. This is not recommended, hence the Eslint rule, but there is no other way to access the accordion element.
-  // Todo: When we upgrade The Design System, we should use the new `Details` component with `getByRole('group')` instead. https://github.com/Altinn/altinn-studio/issues/14577
+const getCodeListDetails = (codeListTitle: string): HTMLElement =>
+  // The following code accesses a node directly with parentElement. This is not recommended, hence the Eslint rule, but there is no other way to access the details element.
```

Comment:
Da har jeg opprettet en sak: https://github.com/digdir/designsystemet/issues/3941
```suggestion
  // The following code accesses a node directly with parentElement. This is not recommended, hence the Eslint rule, but there is no other way to access the details element.
  // Todo: Use getByRole('group') when the role becomes correctly assigned to the component: https://github.com/digdir/designsystemet/issues/3941
```

Reply:
Perfekt!